Sources/OfflineEntitlements/OfflineEntitlementsManager.swift

```diff
@@ -35,6 +35,7 @@ class OfflineEntitlementsManager {
         completion: (@MainActor @Sendable (Result<(), Error>) -> Void)?
     ) {
         guard #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *),
+              !self.systemInfo.observerMode,
```

Comment:
This logic lives both in `OfflineEntitlementsManager` and in `OfflineCustomerInfoCreator`. It would be great to unify it, but probably not a blocker.